dashboard/app/models/levels/gamelab.rb

```diff
@@ -48,6 +48,7 @@ class Gamelab < Blockly
     helper_libraries
     start_libraries
     libraries_enabled
+    validation_enabled
```

Comment:
Should this also be in gamelab_jr, or did you and Amanda decide to not go forward with Spritelab?

Reply:
gamelab_jr inherits from gamelab, and inherits the `serialized_attrs`. same goes for dancelab with inherits from gamelab_jr.

Reply:
👍 - I was assuming the ruby inheritance works the same way as the JS inheritance. Haha remind me never to assume :)